Automattic/wp-calypso
# Stats: Remove the repetitive "View" icon and text per row

**Changes proposed in this Pull Request:**
* Remove the repetitive "View" icon (and text) per row in "Posts & Pages" stats module.
* I also had to fix some ESLinter errors to be able to commit .

**Testing instructions:**

1. Navigate to https://wordpress.com/stats/day/{site}
2. Verify that the external link icon is not there in the "Posts & Pages" module. Each row should be linking to detailed stats page. 
Before:
![image](https://user-images.githubusercontent.com/4550351/57060762-dfdf7480-6cfd-11e9-93af-84a163c45614.png)

After:
![image](https://user-images.githubusercontent.com/4550351/57060772-eb32a000-6cfd-11e9-8b2e-cda4a57fd26a.png)

3. Click "View all" at the bottom of "Posts & Pages" module to go to detailed stats page - verify that "View" link isn't there. 
Before: 
![image](https://user-images.githubusercontent.com/4550351/57060825-17e6b780-6cfe-11e9-8883-e8ed1fab52d1.png)

After:
![image](https://user-images.githubusercontent.com/4550351/57060837-26cd6a00-6cfe-11e9-929a-e31fea56eea6.png)

4. Verify the same for "Videos", and "Authors" module (both use the same list view as "Posts & Pages")
5. Take a look at other modules in Stats and verify that everything is working, and nothing is visually off (I might have inadvertently broken something when working on the linter errors)

See #32252
